.github/workflows/missing_types.yml

```diff
@@ -0,0 +1,29 @@
+name: Missing Types
+
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
I think that it should be triggered only when a PR is merged.

Reply:
Yes, make sense :)

I just followed the issue description. @Eomm do you agree on this?

Reply:
Yes it does

- we add the lebel
- we merge the PR
- the issue is created

we will avoid to open an issue for those PRs that are not merged

Reply:
PR updated to handle this.